Languages.fr.resx

```diff
@@ -0,0 +1,14 @@
+﻿<root>
```

Comment:
I assume dotnet uses the default language as fallback when no override is provided?

Reply:
Everything without a valid translation will be displayed with the default language, yup.